submissions/DawoduRahmon/html-form/contact.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
Why do you have 2 html-form folder? You should remove one of the folders and its content. @Dawodurahmonn